### PR TITLE
Adding hex.win - extremely damaging and ongoing

### PR DESCRIPTION
**Pull request by hex-win**
https://www.theblockcrypto.com/post/49989/the-hex-token-sale-may-be-an-unregistered-securities-offering-but-its-no-ponzi

> It's like saying "Heroin is not a schedule I controlled substance.  There aren't actually any drugs, they're just molecules in the universe."

You can debate:
- Scam? 
- Ponzi? 
- Pyramid scheme?

*(definitely not "certificate of deposit")*

Check the source code: https://etherscan.io/address/0x2b591e99afe9f32eaa6214f7b7629768c40eeb39#code

# `FLUSH_ADDR` ?

Screenshot 2019-12-19 at 13 17 00

_(do your own research where the "released value" is going - **congratulations** - you've just released yourself from ETH coin)_

Some further discussion:
https://www.reddit.com/r/ethfinance/comments/e1zhyj/warning_beware_of_richard_hearts_hex_project/

# ~Use your own brain.~ Actually I'm realising life is complicated and not everyone has time to run deep analysis of everything.

Using throw away account as I want to protect users, as well as my own family - definitely don't want to engage with these type of guys:

Screenshot 2019-12-19 at 13 23 25

If you are a journalist looking for a comment - comment in this thread and I'll try to check again in some time...

### CODE REVIEW

**Comment by hex-win** [comment]
Nice thread: https://twitter.com/TooWumboToFail/status/1204425137371484161

Great article: https://medium.com/@TooWumboToFail/under-a-hex-396847b86e57

> I have seen several articles that call HEX a Ponzi or Pyramid Scheme, both of which it isn’t. It has many features of these, but is unique in that it has so many moving parts, and complexities that these have helped to mask just how unethical a project it is. 

> Even in a sea of ethically dubious projects, HEX stands out as being the most egregious, though **perhaps the most clever**.

> When it comes to the Adoption Amplifier, Richard is careful to avoid saying that you are purchasing HEX for Ethereum. As he terms it, you are “donating” your Ethereum which is then being “transformed” into HEX. To cut through the confusion this language might cause, what is actually happening is that the ETH is not really being “transformed” in any meaningful sense. It still exists, it just doesn’t belong to you anymore, it belongs to Richard. 

> What you will see are a dizzying list of bonuses you can earn as an early staker of HEX, and promises of massive future returns, but no mention of the Origin Address, its implications, or the fact that Richard owns it.

> Even the name “Origin Address” sounds benign and boring. Perhaps if it was called the “Richard Gives Himself Almost Half Of All HEX” Address it would garner more attention.

> If this sounds confusing and convoluted, it is designed to be. All the complexity and game theory serves not only to hook people in, but to disguise the full extent of Richard’s self-enrichment. 

As mentioned in the initial post, it's debatable:
- Scam?
- Ponzi?
- Pyramid scheme?

**perhaps the most clever** - emphasis mine, it's most certainly very clever. Hex is innovative in many ways, does not follow already existing definition of a fraud or a scam.

Mark Twain quote

### "freakout" warning

Out of a sudden, 1000 internet trolls coming at you?

Let's stick to the truth, facts, reality. 

But how can you define real?

*(one more snippet from the article linked at the beginning)*

> From that, what I hope will occur is a more bottom-up emergence of norms around how we treat scams, and those that deal in them. HEX could ultimately be a value by serving as a schelling point for this industry — the thing everyone can agree is a scam, and converge around, limiting the fallout.

Luckily, we are in the early days of decentralisation, everyone uses Metamask, Metamask can protect their users.

We also have some clever dispute resolution running on the blockchain.

### 1️⃣ Kleros
Vitalik shilling: https://twitter.com/vitalikbuterin/status/1207344008306937857

### 2️⃣ Aragon
Almost ready: https://twitter.com/aragonproject/status/1207750977358639104

We could create an integration - disputable cases it is not a job of the repo maintainer but the Court can decide... 

_(this case is not disputable, it just does not fit into the existing definition of a fraud, ponzi, scam, pyramid - it's a little bit smarter but calling investment a donation does not make it legit)_

### in the meanwhile https://www.sec.gov/tcr

image

* https://twitter.com/RichardHeartWin/status/1202360075400237056
* http://archive.is/hLVhj

image

And BTW here is my [referral link](https://github.com/MetaMask/eth-phishing-detect/pull/3498), nope nope, just kidding :)

**Comment by 409H** [comment]
This is the follow criteria that I keep to when issuing a blacklist.

1. Impersonating popular crypto services.
2. Impersonating popular crypto personalities.
3. Sending user-input private keys/secrets to backend services.
4. Conducting a giveaway that requires participant's pay an advance fee.
5. Preying on, or lying to, investors of a project which can be proved without doubt.
6. Pyramid scheme-like language.

I think hex.win is stepping on the toes of points 4, 5, and 6. Let's discuss more.

**Comment by hex-win** [comment]
### These are real screenshots of screenshots

### `66x`
https://t.me/HEXnews/466
Screenshot 2019-12-25 at 11 44 56

### `8x` 
https://t.me/HEXnews/484
Screenshot 2019-12-25 at 11 44 24

### `684.51%`
https://t.me/HEXnews/514
Screenshot 2019-12-25 at 11 43 23

### The reality

https://coinmarketcap.com/currencies/hex/
image

https://www.bidesk.com/en_US/trade/HEX_ETH
Screenshot 2019-12-25 at 11 58 36


image

You can send `1 ETH` and "transform" it to obtain `850000 HEX`.
At `0.00000082` per ETH it's `0.697 ETH`
Instant 30% loss + gas fees + transaction fees + withdrawal fees.

### Great deal, definitely not fraudulent

As I said - we can debate the definition of a scam, fraud, ponzi, pyramid.

And then we can debate further - _"he said it is a donation so legally, technically, theoretically OK"_.

Establishing new definition:
- “As a user you would be much better off without it”
- “Provides profit to the founder only”
- "A well informed person with full information would stay away"

People gambling in casinos are aware what is the establishment: it is a casino, it is a gambling, you can go into debt and borrow money, lose home, lose marriage - fair game. I believe in free will and responsibility for decisions but here the situation is different.

**Comment by 409H** [comment]
Hi, very good points. I will note down. I've started an internal discussion

Please note that I do not work for MetaMask with a contract (I do not represent MetaMask). I have started a discussion with them also

**Comment by hex-win** [comment]
Preserving the evidence for the law enforcement.

* https://twitter.com/cryptoscamhub/status/1202986112265773057
* https://archive.is/LfEap
* https://web.archive.org/web/20191227032925/https:/twitter.com/cryptoscamhub/status/1202986112265773057
* [Peter McCormack.mp4.zip](https://github.com/MetaMask/eth-phishing-detect/files/4004047/Peter.McCormack.mp4.zip) - cannot attach video, can attach .zip with the video

Code does not lie:

* HEX code: https://etherscan.io/address/0x2b591e99afe9f32eaa6214f7b7629768c40eeb39#code
* FLUSH_ADDR: https://etherscan.io/address/0xDEC9f2793e3c17cd26eeFb21C4762fA5128E0399

Some ETH from FLUSH_ADDR moved, might be an interesting "chain-analysis" challenge for holiday season, just don't wait another week to decide if this is scam, fraud, ponzi or pyramid.

If I am a scammer, and I call my scam a "donation" somewhere in the small print on page 14 - that makes me even more scammer.

**Comment by hex-win** [comment]
> ...but Richard Heart, who created the contract says nobody needs to know who that is.

> Who could it be? So mysterious. Truly we may never know.

via Reddit https://www.reddit.com/r/CryptoCurrency/comments/eff5y9/following_the_hex_scam_just_under_800_eth_have/

It gets some coverage, the scam / fraud / ponzi / pyramid / HYIP / pump-and-dump is ongoing. Credit to Richard, he is so likeable and professional when scamming people!

Screenshot 2019-12-29 at 10 49 22

**Comment by hex-win** [comment]
https://twitter.com/RichardHeartWin/status/1212325916568891392
image

This is a quality piece of reverse psychology. Richard is able to manipulate the audience using phrases like:

> "maybe it looks like a scam, but in order to spread virally it has to use scammy techniques"

And then it is possible to think:

> "OK, he says it's not a scam, only a scammy technique, so OK" 

------


If I was a SEC or any other financial crime agency working in the US, I would allow this scam to be as damaging as possible - bad name for crypto, bad name for Bitcoin and Etherium _(terrorism or child pornography anyone?)_, more chances to keep USD as the world reserve currency.

I’m in the crypto space because blockchain is one of the best chances of humanity to survive - new models of trust, ownership, governance.

These type of frauds are dangerous.

In the aftermath of this event, I suggest making a new law:

## Promoting Ponzi Law

It will make anything think twice before engaging in a fraudulent operation.

Always DYOR.

Promoting a scam makes you a scammer.

Golden rule of investing - always DYOR.

Therefore by promoting the scam you a scammer.

Being a victim is not additionality penalised - only the promoting and reaping benefits.

**Comment by hex-win** [comment]
### Crypto Fundraising Oracle Decentralisation

https://cryptounlocked.wetrust.io/will-hexcrypto-by-richardheartwin-be-blacklisted-by-metamask-io-maintainers-before-29-feb-2020-11-59pm-utc-00-00--6qMCg041BA

image

**Comment by hex-win** [comment]
https://decrypt.co/15025/cryptos-101-stupidest-moments-of-2019-october-december

image

**Comment by hex-win** [comment]
* https://twitter.com/davidgerard/status/1216288172310704128
* https://cryptoslate.com/hex-origin-address-emptied-out-almost-7-million-withdrawn/

image

Why withdrawing money so early if the scam is successfully ongoing?

This is the most puzzling piece... 🤷‍♀️

**Comment by tarmo888** [comment]
@hex-win you are impersonating popular crypto service with the username that you have picked.
The one who is extremely damaging here is you because you are chicken to do it under your own name.

Hex.win domain is not a phishing site, it is legitimate project built on Ethereum blockchain. The project code is verified on Ethersacan and the project is explained in layman terms line by line on the website.

**Comment by tarmo888** [comment]
@409H if hex.win gets censored then eos.io should had been censored too when it had its crowdsale on Ethereum. The crowdsale of HEX is not much different than crowdsale of EOS.

But there is a big difference, while EOS gave you worthless ERC20 tokens and let you generate a private key on a non-existing new blockchain, with HEX project you mint yourself (execute a function)  new ERC20 with the MetaMask wallet (or any other Ethereum wallet).

HEX ERC20 token is the final product. Everything that this project will does is already locked into the smart-contract code, there is no promise of any new blockchain to be built with the ETH that collected. It is flat out said that you will not get any benefit from that ETH.

EOS had promises and they settled with SEC, HEX is a working product already today.

**Comment by hex-win** [comment]
> because you are chicken to do it under your own name

Protecting identify of myself and my family.

Facts remains facts regardless my race, religion, veteran status.

>  It is flat out said that you will not get any benefit from that ETH.

That's the point.

Your comment preserved as screenshot: https://imgur.com/Rm7JBfH 

(would be nice to use https://tlsnotary.org/ to prove it was authentic)

https://twitter.com/RichardHeartWin/status/1191860770172231681/hidden
image

## It is flat out said that you will not get any benefit from that ETH

**Comment by tarmo888** [comment]
@hex-win You are using a popular crypto projects name as your username, so you are not just protecting your identity, you are confusing people. You are the scammer here.

You have some personal dislike for the project, maybe you FOMOd in on the first day and later figured out that you could get more if you spread out over days, who knows. Stop harassing legit projects just because your personal dislike, there are many people on Telegram who like the project and have bothered to learn how it works.

You are now posting with sarcasm, which is not helpful either. Besides that, you don't seem to understand the difference of words "can/could" and "will". The success of the project is in the hands of the community, not the who launched it. And the community isn't promising anything either, everybody just builds more tools for it. As decentralized as one true cryptocurrency should be.

**Comment by hex-win** [comment]
I'm a little bit autistic _(think Asperger, that Swedish girl screaming "How dare you")_ - I do not engage sarcasm, irony or rhetorical questions.

You said: 

> you are now posting with sarcasm

I repeat, there is no such thing as sarcasm in my words.

What I did, was to contrast:

> **1.** It is flat out said that you will not get any benefit from that ETH
> **2.** Is the best opportunity I've ever seen in my entire life. I think this can make millions of people truly wealthy.

No personal dislike. 

Quite the opposite, I truly respect Richard and his ability to scam, defraud, manipulate people.

# Simple question

https://www.sec.gov/tcr

image

Assuming it is not a securities fraud and (hypothetically the project is legit), is there any way for me to make any profit?

Other than: https://en.wikipedia.org/wiki/Greater_fool_theory

# 🤷‍♀️

**Comment by tarmo888** [comment]
@hex-win 

**Ponzi scheme?**
* Does it pay interest to previous investors with the investments money of new investors? No, the interest is paid from inflation, which is controlled by smart-contract.
* Will it be unable to pay the promised interest in case more people want to take money out than there is new people putting in? No, the interest is paid from inflation, which is controlled by smart-contract.
* Are there unlicensed sellers of HEX? No, you can mint your own HEX with the smart-contract, nobody is sending you any HEX when you send ETH to the address, you have to execute the function yourself to make the smart-contract to create new HEX.
* Are the strategies how to use HEX secret? No, everything is publicly available on the website and discussed to death on Telegram channels.
* Is there difficulties to receive the promised HEX? No, no human being controls that, it is all done by smart-contract, it cannot fail to transfer you the funds that it is suppose to send you.

**Pyramid scheme?**
* Is there multi-level referral system? No, only 1 level like many other legit businesses have.
* Can the Pyramid collapse? No, because there is no multi-level system there.

**High-Yield Investment Programs**
* Does it promise you high returns in USD, BTC or ETH? No, but it says it CAN happen. That's not promise.
* Does it promise you high returns in its own currency HEX? Yes, and it will not fail doing that because this is hard-coded into the smart-contract, which code says that if you stake during the first year, you get lot more HEX. It simply cannot fail that promise because there is nobody who could stop it because it is written into the immutable contract. This is no different from Bitcoin, which also has hard-coded high inflation during the first 4 years before the halving.

Greater fool theory applies to all cryptocurrencies and even stock markets. If you buy any cryptocurrency, somebody is selling it to you to get out of their position.

**Comment by hex-win** [comment]
https://github.com/MetaMask/eth-phishing-detect/pull/3498#issuecomment-568891822

### As I said - we can debate the definition of a scam, fraud, ponzi, pyramid.

Typical technique of well-executed scam.

Debating nitty-gritty semantics of the word "is":

### https://www.youtube.com/watch?v=BS-cip2brsg

> It depends on what the meaning of the word 'is' is.

**Comment by tarmo888** [comment]
@hex-win Nonsense, the definition of Ponzi scheme and Pyramid scheme are very clear, no need to debate that.

Scam and fraud are pretty clear too, it involves tricking somebody by withholding some information.

HEX smart-contract source code is publicly available on Etherscan and there is even Laymen's explanation of what each function does.

HEX is NOT collateralized nor pegged with ETH, so you are NOT tricked out of money either. There is NO promised to get that back, there is opposite promise that you should not expect any benefit from that ETH.

**Comment by hex-win** [comment]
Evidence: https://web.archive.org/web/20200125214149/https://hex.win/
Yes, it is a valid evidence: https://www.theregister.co.uk/2018/09/04/wayback_machine_legit/

> Prosecution had included testimony from the Internet Archive's office manager, who explained how the Archive captures and preserves evidence of the contents of the internet at a given time.
The court decided this provided "sufficient proof" that its mirrored pages were authentic.

Screenshot 2020-01-25 at 22 38 38

> # `HEX IS DESIGNED TO INCREASE IN VALUE FASTER THAN ANYTHING ELSE IN HISTORY.`

https://consumer.findlaw.com/securities-law/what-is-the-howey-test.html

> Under the Howey Test, a transaction is an investment contract if:
> * It is an investment of money
> * There is an expectation of profits from the investment
> * The investment of money is in a common enterprise
> * Any profit comes from the efforts of a promoter or third party

**Comment by tarmo888** [comment]
@hex-win You clearly haven't even made clear what HEX is because this is addressed in so many discussions already. You are not discovering anything new.

* Same way like the words **can/could** are not same as **will**, the word **designed** is not a promise of profit. Bitcoin is also **designed** to be store of value and next reserve currency, but that doesn't mean it **will** be that. Bitcoin has reward halving and fixed supply **designed** into it, HEX has the staked supply and small inflation after the first year of distribution + many other mechanics during the first year that are also **designed**.
* These first 2 Howey tests would fail with every cryptocurrency.
* You don't invest money into common enterprise, you just give the ETH away in order to be able to mint yourself HEX on next day based on percentage it made compared to all last day **investors**. Nobody gives you that privilege, this rule is written into the smart-contract.
* Profits doesn't come from the efforts of a promoter or third party, the ETH is not used to make HEX any better (actually, no promise has been made what the HEX will be used, there is no evidence who it belongs). The profits depend on you, you have to execute the smart-contract to stake and end the stake, you have to sell your coins to make profit. There is no single entity that controls what the smart-contract does, every single actions that the smart-contract does is because users interact with it. Censoring the hex.win website won't change it either because these functions can be execute with MyEtherWallet or Etherscan. hex.win is merely a tool that interacts with Ethereum blockchain. Nobody holds user tokens except the users themselves, so only users can make the profit for themselves.

HEX will not fail the Howey test the same way how EOS.IO did because EOS.IO promised investors to build a software for EOS network when all they had was private key generator and similar crowdsale website, but HEX team already built everything that runs HEX, the network (smart-contract) was already deployed before anybody could use the website. EOS was not a scam, ponzi, pyramid and HEX isn't either. Drop your grudge, you are wasting everyones time.

**Comment by 409H** [comment]
@tarmo888 

> @409H if hex.win gets censored then eos.io should had been censored too when it had its crowdsale on Ethereum. The crowdsale of HEX is not much different than crowdsale of EOS.

That would be incorrect. EOS and HEX promoted/marketed themselves with completely different language. Please don't try to throw other projects into the discussion. We are talking about HEX in this thread.

**Comment by tarmo888** [comment]
@409H you missed the point, HEX can't fail delivering its promises because there is none, there is only examples what is possible by design (written into the smart-contract). EOS was a security, which promised to release a mainnet, which wasn't even done when crowdsale launched. HEX code is done and immutable in smart-contract. Both use same crowdsale. If EOS would have failed to deliver the software, it would have been scam, HEX can't fail that because the end-product is already delivered.

**Comment by 409H** [comment]
> HEX can't fail that because the end-product is already delivered.

You're implying something is not a scam because it is "100% launched". End products _can_ be a scam. The marketing material also made (soft) promises of future gains - semantics, sure... but it did.

**Comment by tarmo888** [comment]
@409H No, I am saying that end product, which is an immutable smart-contract cannot fail its promises. Ignorant people are trying to compare HEX with BitConnect, which failed it's promised because it had centrally controlled trading bot. HEX has none of that and the promises that are about things that are hard-coded into the smart-contract. Gains in other currencies are not promises, but just examples of what could happen. And if you don't believe it then you have source code, Layman's term Google Docs and many hours of Youtube videos, which explain every feature again and again.

Where is a scam in product, which the founders cannot make it to fail its promises? Only MetaMask as a gatekeeper can make it fail, but even MetaMask can't kill it because it works with any Ethereum wallet, just MetaMask moderation is the greatest risk in wounding it and target for haters with nonsense issues like this.

**Comment by hex-win** [comment]
I think one of the rebuttals regarding ponzi / pyramid is a small print somewhere that says “donation”.

-----

This case may set a precedent of a new law:

**Those promoting a scam are also criminally responsible.**

* Richard Heart to be captured by the Interpol (10 years in prison)
* Those in the conspiracy to defraud others are also responsible (up to 2 years in prison for major figures)

**Comment by tarmo888** [comment]
@hex-win whether something is donation or not doesn't make or unmake something a Ponzi/Pyramid scheme, could you please finally learn what the definition of Ponzi/Pyramid is. HEX is neither of them.

By calling everything you personally don't like a "scam", you are not helping anybody, you make it worse for the whole space https://twitter.com/on_yavin/status/1224239622546432000

**Comment by The-WRITTEN-SIGN** [comment]
The nefarious troll "Hex-Win" brings up the Howey Test.
https://www.investopedia.com/terms/h/howey-test.asp

The Howey test determines whether a transaction is, in fact, an example of an "investment contract" defined by US law. This test determines whether an asset is a security.

The Howey test thus is not important in this discussion, since a smart contracts might pass or not pass and still be legitimate. 

https://www.coinspeaker.com/lawsuits-filed-binance-bitmex-block/

The example of Block.One the company involved with launching EOS.

Many well known crypto companies, like Binance, Block.one try to avoid passing the howey test to reduce the amount of regulation they fall under, but the might still pass.

**Comment by The-WRITTEN-SIGN** [comment]
Any sufficiently innovative new crypto project goes through a phase of hazing by the rest of the community.

Maybe it is an evolutionary strategy of a community to test the strength and conviction of a new challenger before trust is established.

The hazing of a project is proportional to the impact and potential of a new project.

The HEX project launched in the beginning of December a finished product and is therefore still quite new. It has currently a market cap  of $190 million and ranks 38 on https://coinpaprika.com/.

Therefore the HEX project endures a lot of trolling and FUD. This attack by "HEX-win" is only one example.

**Comment by The-WRITTEN-SIGN** [comment]
HEX Security Audit by CoinFabrik

https://hex.win/docs/HEX-Security-Audit-by-CoinFabrik-DEC2019.pdf

HEX Economics Audit by CoinFabrik

https://hex.win/docs/HEX%20Economics%20Audit%20by%20CoinFabrik.pdf

**Comment by davidwgrossman** [comment]
I'm going to close this one out as it is more than 1 year old. Feel free to re-open if further discussion is desired.

**Comment by marsrobertson** [comment]
I wish I'd "aped in" back in 2019 rather than being worried about reputational concerns :(

What is happening now?

image

* https://twitter.com/RichardHeartWin/status/1633059907208462336A
* https://archive.ph/Ucaor
